### PR TITLE
Clean up assert usages to use more specific methods

### PR DESCRIPTION
**Pull request by lindner**
- Length checks now use assert.lengthOf
- Length checks for 0 now use assert.isEmpty
- assert() with == now use assert.equal
- assert() with includes switched to assert.include
- checks for instanceof switched to assert.instanceOf
- checks for undefined switched to assert.isUndefined
- checks for > value switched to assert.isAbove

### CODE REVIEW

File: runtime/manual_tests/loader-tests.js [hunk 1]
```diff
@@ -18,19 +18,19 @@ describe('loader', function() {
...
+    assert.equal(manifest.schemas.Restaurant.fields.servesCuisine, 'Text');
   }).timeout(10000);
 
   it('can read a schema.org schema with multiple inheritance', async () => {
     let schemaString = await loader.loadResource('http://schema.org/LocalBusiness');
     let manifest = await Manifest.parse(schemaString, {loader, fileName: 'http://schema.org/LocalBusiness'});
-    assert(manifest.schemas.LocalBusiness.fields.duns == 'Text');
-    assert(manifest.schemas.LocalBusiness.fields.branchCode == 'Text');
+    assert.equal(manifest.schemas.LocalBusiness.fields.duns, 'Text');
+    assert(manifest.schemas.LocalBusiness.fields.branchCode, 'Text');
```
Comment (mmandlis) [inline runtime/manual_tests/loader-tests.js]: `.equal` is missing here

Comment (lindner) [reply runtime/manual_tests/loader-tests.js]: Thanks for spotting that!

Pushed a new change and updated the PR.

**Comment by mmandlis** [comment]
selenium tests are consistently failing for this pr, do they pass locally for you?

**Comment by lindner** [comment]
@mmandlis - my local laptop environment exhibits some strange behavior on head that I haven't been able to diagnose yet.

I'll try to get things a more stable before I send more PRs.

**Comment by lindner** [comment]
PTAL, all cleaned up now.

I now am more aware of which assert library is being used in each file.

**Pull request merged**